Tribler/Test/Core/Upgrade/test_db72_to_pony.py

```diff
@@ -110,11 +111,9 @@ def test_old_db_version_check(self):
         # Wrong old database version
         old_db = os.path.join(self.session_base_dir, 'old.db')
         shutil.copyfile(OLD_DB_SAMPLE, old_db)
-        conn = sqlite3.connect(old_db)
-        with conn:
+        with contextlib.closing(sqlite3.connect(old_db)) as conn, conn:
```

Comment:
Also, use either `connection` or `conn` everywhere. If the stuff should co-exist in the same context, give names like `connection1`, etc.

Reply:
Done